types/options.go

```diff
@@ -49,6 +49,12 @@ var (
 	defaultConfigFile = SystemConfigFile
 	// DefaultStoreOptions is a reasonable default set of options.
 	defaultStoreOptions StoreOptions
+
+	// defaultOverrideConfigFile path to override the default system wide storage.conf file
+	defaultOverrideConfigFile = "/etc/containers/storage.conf"
+
+	// defaultDropInConfigDir path to the folder containing drop in config files
+	defaultDropInConfigDir = "/etc/containers/storage.conf.d"
```

Comment:
Let's make those `const` if they're not variable.

Reply:
Because of this change https://github.com/containers/storage/pull/1885#discussion_r1581176041, we may not be able to make it a `const`.